.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUSTFLAGS: --deny warnings
+
+jobs:
+  style:
+    name: 'Check basic style'
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Check basic style
+      run: ./.github/check-basic-style.sh
+
+  check:
+    name: 'Build & test'
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        rust:
+          # - stable   # add again on oct 8th
```

Comment:
This PR is still blocked by this. Will update as soon as the next stable is released.